### DISCUSSION TITLE
React Preview for WebStorm  / IntelliJ

### DISCUSSION THREAD

**Discussion by fwouts** [fwouts/previewjs#2] [Ideas]
⬆️ Upvote this idea if you'd like to see React Preview released for WebStorm and IntelliJ.

**Comment by johnfrades** [comment]
yes please, thanks!

**Comment by datner** [comment]
Would be quite neat

**Comment by gilbarbara** [comment]
I was using it regularly but the current plugin version doesn't work with Webstorm 2021.3.

Can you release a new version for it?

**Reply by fwouts** [reply]
Thanks for letting me know! I'll try and get a new release out but might not be until a few days because of the holiday season 🙂

**Reply by fwouts** [reply]
Hi Gil, sorry it took more than a few days, and happy new year! You should find the new version (v0.7.1) very soon in the JetBrains store, as soon as they've approved it. Let me know how you go!

**Reply by gilbarbara** [reply]
Hey @fwouts 
It's available now as 0.7.2.

Thanks!

**Reply by mallowigi** [reply]
Just installed it and it doesn't work. I tried many projects, nothing.

Do I have to install something prior to using the plugin?

**Reply by gilbarbara** [reply]
hey @mallowigi 
Install the Preview.js plugin instead

**Reply by fwouts** [reply]
Heads-up that there was a bug in Preview.js that prevented it from working with non-TypeScript projects, v1.0.4 is awaiting approval and will fix that hopefully later today.

In the meantime, you can add @types/react as a dependency to your project and it should fix it 🙂

**Reply by fwouts** [reply]
Update: v1.0.4 is now available on the JetBrains Marketplace: https://plugins.jetbrains.com/plugin/18384-preview-js
